src/DataTypes/convertYTsaurusDataType.cpp

```
@@ -0,0 +1,399 @@
+    }
+    else if (data_type == "uint64")
+    {
+        data_type_ptr = DataTypeFactory::instance().get("UInt64");
+    }
+    else if (data_type == "uint32")
+    {
+        data_type_ptr = DataTypeFactory::instance().get("UInt32");
+    }
+    else if (data_type == "uint16")
+    {
+        data_type_ptr = DataTypeFactory::instance().get("UInt16");
+    }
+    else if (data_type == "uint8")
+    {
+        data_type_ptr = DataTypeFactory::instance().get("UInt8");
+    }
+    else if (data_type == "float")
+    {
+        data_type_ptr = DataTypeFactory::instance().get("Float");
+    }
+    else if (data_type == "double")
+    {
+        data_type_ptr = DataTypeFactory::instance().get("Float64");
+    }
+    else if ((data_type == "boolean" && !type_v3) || (data_type == "bool" && type_v3))
+    {
+        data_type_ptr = DataTypeFactory::instance().get("Bool");
+    }
+    else if (data_type == "string")
+    {
+        data_type_ptr = DataTypeFactory::instance().get("String");
+    }
+    else if (data_type == "utf8")
+    {
+        data_type_ptr = DataTypeFactory::instance().get("String");
+    }
+    else if (data_type == "json")
+    {
+        data_type_ptr = DataTypeFactory::instance().get("JSON");
+    }
+    else if (data_type == "uuid")
+    {
+        data_type_ptr = DataTypeFactory::instance().get("UUID");
+    }
+    else if (data_type == "date32")
+    {
+        data_type_ptr = DataTypeFactory::instance().get("Date");
+    }
+    else if (data_type == "datetime64")
+    {
+        data_type_ptr = DataTypeFactory::instance().get("DateTime64(0)"); // In seconds
+    }
+    else if (data_type == "timestamp64")
+    {
+        data_type_ptr = DataTypeFactory::instance().get("DateTime64(6)"); // In microseconds
+    }
+    else if (data_type == "interval64")
+    {
+        data_type_ptr = DataTypeFactory::instance().get("Int64");
+    }
+    else if (data_type == "date")
+    {
+        data_type_ptr = DataTypeFactory::instance().get("Date");
```

Are they still broken? Lets throw an exception